doc/source/examples/sharp-immersed-boundary-solver/sedimentation-64-particles/sedimentation-64-particles.rst

```diff
@@ -0,0 +1,349 @@
+
+* The ``initial refinement`` is set to 3. Here we want to have the mesh as small as possible for the first time step around each of the particles. To achieve this, we refine every element with at least one vertex in the refinement zone around the particle 6 times before the simulation starts. This ensures that all the cells in the refinement zone around the particle is as small as possible.
+
+* The ``fluid density`` is set to 0.001 according to the description of the problem. As mentioned above, this parameter is a duplication of the density parameter in the physics properties. This will be changed soon, and this parameter will be removed.
+
+* The ``integrate motion`` is set to true because we are interested in the dynamic of the particle as it sediments in the rectangular box.
+
+* The ``assemble Navier-Stokes inside particles`` is set to false because we are not interested in the flow inside of the particle.
+
+* The ``length ratio`` as been set to 2. This is small enough so it does not impact too much the conditioning of the matrix while avoiding interpolation of the immersed boundary stencil in multiple elements.
+
+* The ``contact search radius factor`` is set to 1.5. This parameter is smaller than the default one since the particle motion relative to their size is relatively slow. This enables the use of a smaller search radius which increases the DEM calculation speed.
+
+* The ``particle nonlinear tolerance`` has been set to 1e-3. This is small enough to ensure that the particle dynamics are adequately resolved. We expect a velocity of the particle of the order of 10.
+
+* The ``DEM coupling frequency`` is set to 1000. This is the number of DEM time steps performed per CFD time step. Here 1000 is enough to prevent instability due to particles' contact.
+
+* The ``enable lubrication force`` is set to true since the subgrid lubrication force model is required to capture the lubrication force between the particle when the gap between them is inferior to two times the mesh size.
+
+* The ``lubrication range max`` is set to 2. The subgrid lubrication force model is enabled when the gap between the particle is smaller than two times the mesh size.
+
+* The ``lubrication range min`` is set to 0.1. The subgrid lubrication force model minimal gap considered between the particle is 0.1 times the mesh size.         
+
+* The ``load particles from file`` is set to true to enable the particle to be defined using an external file.
+
+* The ``particles file`` is set to ``particles.input``, which is the file where the particles are defined.
+
+* The ``gravity`` ``Function expression`` is set to 0;0;-981 according to the definition of the case. As we choose the long axis of the rectangular box along the Y, we define gravity in this direction. 
+
+.. note:: 
+    * The number of particles is not define since here the particles are defined using a file. In this case the number of particles is define by the number of particles defined in the file.
```

Comment:
```suggestion
    * The number of particles is not defined here since the particles are defined using a file. In this case the number of particles is defined by the number of particles defined in the file.
```